AC: Added processing of device configs for composite devices (#1434)
openvinotoolkit/open_model_zoo: tools/accuracy_checker/accuracy_checker/launcher/dlsdk_launcher_readme.md

* `gpu_extensions` (path to extension *.xml file with OpenCL kernel description for gpu).
* `bitstream` for running on FPGA.

For setting device specific flags, you are able to use `-dc` or `--device_config` command line option. Device config should be represented as YML file with dictionary of one of two types:

> 1. keys are plugin configuration keys and values are their values respectively

Reviewer: ```suggestion
1. keys are plugin configuration keys and values are their values respectively. In this way configuration will be applied to current running device
```

> 2. keys are supported devices and values are dictionaries with plugin configuration keys and their values

Reviewer: ```suggestion
2. keys are supported devices and values are plugin configuration dictionaries for each device. Plugin configuration represented as dictionary where keys are plugin configuration keys and values are their values respectively
```

Each supported device has own set of supported configuration parameters which can be found on device page in [Inference Engine development guide](https://docs.openvinotoolkit.org/latest/_docs_IE_DG_supported_plugins_Supported_Devices.html)

**Note:** Since OpenVINO 2020.4 on platforms with native bfloat16 support models will be executed on this precision by default. For disabling this behaviour, you need to use device_config with following configuration:
```yml

> ENFORCE_BF16: "NO"

Reviewer: for possibility to use in composite device
```suggestion
CPU:
   ENFORCE_BF16: "NO"
```
